python/dgl/graphbolt/impl/torch_based_feature_store.py

```diff
@@ -23,37 +23,45 @@ def __init__(self, torch_feature: torch.Tensor):
         ----------
         torch_feature : torch.Tensor
             The torch feature.
+            Note that the dimension of the tensor should be greater than 1.
 
         Examples
         --------
         >>> import torch
-        >>> torch_feat = torch.arange(0, 5)
-        >>> feature_store = TorchBasedFeature(torch_feat)
+        >>> from dgl import graphbolt as gb
+        >>> torch_feat = torch.arange(10).reshape(2, -1).to("cuda")
+        >>> feature_store = gb.TorchBasedFeature(torch_feat)
         >>> feature_store.read()
-        tensor([0, 1, 2, 3, 4])
-        >>> feature_store.read(torch.tensor([0, 1, 2]))
-        tensor([0, 1, 2])
-        >>> feature_store.update(torch.ones(3, dtype=torch.long),
-        ... torch.tensor([0, 1, 2]))
-        >>> feature_store.read(torch.tensor([0, 1, 2, 3]))
-        tensor([1, 1, 1, 3])
+        tensor([[0, 1, 2, 3, 4],
+                [5, 6, 7, 8, 9]])
+        >>> feature_store.read(torch.tensor([0]))
+        tensor([[0, 1, 2, 3, 4]])
+        >>> feature_store.update(torch.tensor([[1 for _ in range(5)]]),
+        ...                      torch.tensor([1]))
+        >>> feature_store.read(torch.tensor([0, 1]))
+        tensor([[0, 1, 2, 3, 4],
+                [1, 1, 1, 1, 1]])
 
         >>> import numpy as np
         >>> arr = np.arange(0, 5)
         >>> np.save("/tmp/arr.npy", arr)
-        >>> torch_feat = torch.as_tensor(np.load("/tmp/arr.npy",
-        ...         mmap_mode="r+"))
-        >>> feature_store = TorchBasedFeature(torch_feat)
+        >>> torch_feat = torch.as_tensor([np.load("/tmp/arr.npy",
```

Comment:
converting 1-D array to multi-dim via `[]` is common? It looks weird to me. Or this example aims to demonstrate how to load a on-disk numpy array. If yes, it's better to use a 2-D numpy array instead of `np.arange`.